network-config-analyzer/CLExecute.py

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright 2020- IBM Inc. All rights reserved
+# SPDX-License-Identifier: Apache2.0
+#
+
+import sys
+from os import path
+
+from OutputConfiguration import OutputConfiguration
+from PeerContainer import PeerContainer
+from NetworkConfig import NetworkConfig
+import NetworkConfigQuery
+
+
+class CLExecute:
+    """
+    A class for running queries from commandline
+    """
+    def __init__(self, ns_list='', pod_list='', output_format='txt', output_path=None):
```

Comment:
When we do not specify the flag output_format in the cli args, it sends a None value, and the default 'txt' value is not applied. So currently the OutputConfiguration object is created with outputFormat = None and we get the error : 
"unexpected outputFormat in output configuration value [should be txt or yaml],  value is: None".

Reply:
set a default value `txt` for the `--output` arg.